ff2mpv.js

```diff
@@ -1,7 +1,19 @@
 function ff2mpv(url) {
+    notifyContentScript();
     browser.runtime.sendNativeMessage("ff2mpv", { url: url });
```

Comment:
I just realized that since we already have the `activeTab` permission, the content script could be done away with and the pausing implemented simply (and rather crudely) as 
```
browser.tabs.executeScript({
    code: "video = document.getElementsByTagName('video');video[0].pause();"
});
```
before sending the url to the native client. 
What do you think @woodruffw ?

Reply:
Hah, I had no idea you could do that. That's scary.

But yeah, lets go ahead and (ab)use that feature.